stream-chat-android-client/src/main/java/io/getstream/chat/android/client/experimental/plugin/listeners/QueryChannelsListener.kt

```diff
@@ -1,13 +1,34 @@
 package io.getstream.chat.android.client.experimental.plugin.listeners
 
+import io.getstream.chat.android.client.ChatClient
 import io.getstream.chat.android.client.api.models.QueryChannelsRequest
 import io.getstream.chat.android.client.models.Channel
 import io.getstream.chat.android.client.utils.Result
 import io.getstream.chat.android.core.ExperimentalStreamChatApi
 
 @ExperimentalStreamChatApi
+/**
+ * Listener of [ChatClient.queryChannels] requests.
+ */
 public interface QueryChannelsListener {
+
+    /**
+     * Run precondition for the request. If it returns [Result.success] then the request is run otherwise it returns
+     * [Result.error] and no request is made.
+     */
```

Comment:
Can we follow the 

@param ...
...
@return notation here for the docs?

Reply:
We have many places to update. Thanks for pointing it out.